pkg/storage/tpr/storage_interface_test.go

```diff
@@ -284,6 +284,57 @@ func TestGetList(t *testing.T) {
 	}
 }
 
+func TestUpdateNonExistent(t *testing.T) {
```

Comment:
I'm reading this as meaning that updating a non-existent object (at this level) is doable with TPR -- is that accurate?  If so I think an explanatory comment would pay dividends here, otherwise LGTM.